src/main/java/org/jalasoft/moi/model/core/ICacheProvider.java

```diff
@@ -0,0 +1,16 @@
+package org.jalasoft.moi.model.core;
+
+/**
+ * Provides a cache format
+ *
+ * @author Lucero Quiroga Perez
+ * @version 1.1
+ */
+public interface ICacheProvider {
+
+    Process getProcessById(long pid);
+
+    void add(long pid, Process process);
+
+    void deleteProcess(long pid);
+}
```

Comment:
We should stablish if interfaces are going to be documented or just when a class implements an interface an document the overriding.